source/crud/tests/unified/updateOne-sort.yml

```diff
@@ -0,0 +1,96 @@
+description: updateOne-sort
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: client0
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+  - database:
+      id: database0
+      client: client0
+      databaseName: crud-tests
+  - collection:
+      id: collection0
+      database: database0
+      collectionName: coll0
+
+initialData:
+  - collectionName: coll0
+    databaseName: crud-tests
+    documents:
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+
+tests:
+  - description: UpdateOne with sort option
+    runOnRequirements:
+      - minServerVersion: "8.0"
+    operations:
+      - name: updateOne
+        object: collection0
+        arguments:
+          filter: { _id: { $gt: 1 } }
+          sort: { _id: -1 }
+          update: { $inc: { x: 1 } }
+        expectResult:
+          matchedCount: 1
+          modifiedCount: 1
+          upsertedCount: 0
+    expectEvents:
+      - client: client0
+        events:
+          - commandStartedEvent:
+              command:
+                update: coll0
+                updates:
+                  - q: { _id: { $gt: 1 } }
+                    u: { $inc: { x: 1 } }
+                    sort: { _id: -1 }
+                    multi: { $$unsetOrMatches: false }
+                    upsert: { $$unsetOrMatches: false }
+          - commandSucceededEvent:
+              reply: { ok: 1, n: 1 }
+              commandName: update
+    outcome:
+      - collectionName: coll0
+        databaseName: crud-tests
+        documents:
+          - { _id: 1, x: 11 }
+          - { _id: 2, x: 22 }
+          - { _id: 3, x: 34 }
+
+  - description: updateOne with sort option unsupported (server-side error)
+    runOnRequirements:
+      - maxServerVersion: "7.99"
+    operations:
+      - name: updateOne
+        object: collection0
+        arguments:
+          filter: { _id: { $gt: 1 } }
+          sort: { _id: -1 }
+          update: { $inc: { x: 1 } }
+        expectError:
+          isClientError: false
+    expectEvents:
+      - client: client0
+        events:
+          - commandStartedEvent:
```

Comment:
I don't recall that being done in other tests. Seems like it'd be more a test of APM than this functionality, since `expectError.isClientError: false` covers the check for a server-side error already.